MetasysServices/Audits/AuditFilter.cs

```diff
@@ -10,30 +10,45 @@ namespace JohnsonControls.Metasys.BasicServices
     public class AuditFilter : TimeFilter
     {
         /// <summary>
-        /// Filter by comma-separated list of origin applications.
-        /// See /enumSets/578/members for possible values.
-        /// Example: 1,2.
+        /// Holds enum value of origin applications.
         /// </summary>
-       public string OriginApplications { get; set; }
+       public IEnumerable<OriginApplicationsEnum> OriginApplicationsEnum { get; set; }
 
+        /// <summary>
+        /// Filter by comma-separated list of origin applications.
+        /// See /enumSets/578/members for possible values.
+        /// Example: 1,2.
+        /// </summary>
+        public string OriginApplications { get; set; }

```

Comment:
```suggestion
        public string OriginApplications { get; set; }
the string property needs to be removed and managed internally before sending it to the server
```

Reply:
The Audit API accepts a string as OriginApplications, ClassLevels, and ActionTypes, If I try to pass enum instead of string it throws an exception when send to the server.

Reply:
The enum needs to be converted in a string and injected into the dictionary key before sending the request to the server.